data/random-teams.ts

```diff
@@ -41,9 +41,9 @@ const SpecialSetup = [
 const MixedSetup = [
 	'clangoroussoul', 'growth', 'happyhour', 'holdhands', 'noretreat', 'shellsmash', 'workup',
 ];
-// Moves which boost Speed:
+// Moves which only boost Speed:
 const SpeedSetup = [
-	'agility', 'autotomize', 'flamecharge', 'rockpolish', 'shiftgear',
+	'agility', 'autotomize', 'flamecharge', 'rockpolish',
 ];
```

Comment:
If this actually means something now, and isn't just a hardcode, Aura Wheel and Rapid Spin should be included.

Reply:
this goes across all gens. rapid spin is excluded for that reason, since it'd mess with oldgens otherwise. plus, some mons want sr+spin even in gen 8. Rapid Spin doesnt "only boost Speed", it also removes hazards.

i'll include aura wheel though, shouldn't affect anything too much? i think

Reply:
never mind, adding aura wheel messes with morpeko's moveset a lot actually. parting shot is rejected with speedsetup